services/ui-src/src/routes/Intro/Intro.tsx

```diff
@@ -1,29 +1,83 @@
 // components
-import { Box, Heading, Text } from "@chakra-ui/react";
-import { ReportPage } from "components";
+import { Box, Button, Image, Heading } from "@chakra-ui/react";
+import { BasicPage, Section } from "components";
+import { useNavigate } from "react-router-dom";
```

Comment:
Nit: Not a component, so should be on line 1 above components to match other files.